src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Receiving shared data
+subhead: Sharing on mobile made simple
+authors:
+  - petelepage
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  On a mobile device, sharing should be as simple as clicking the Share button,
+  choosing an app, then choosing who to share with. The Web Share Target API
+  allows installed web apps to register with the underlying operating system to receive shared content.
+tags:
+  - post
+  - capabilities
+---
+
+On a mobile device, sharing should be as simple as clicking the **Share** button,
+choosing an app, and choosing who to share with. For example, you may want to
+share an interesting article, either by emailing it to friends or tweeting it to
+the world.
+
+In the past, only native apps could register with the operating system to
+receive shares from other installed apps. But with the Web Share Target API,
+installed web apps can register with the underlying operating system
+as a share target to receive shared content.
+Support for text and data was added in Chrome 71, and
+support for files was added in Chrome 76.
+
+{% Aside %}
+The Web Share Target API is only half of the magic. Web apps can share data,
+files, links, or text using the Web Share API. See
+[Share like a native](/share-like-a-native) for details.
+{% endAside %}
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 400px;" alt="Android phone with the 'Share via' drawer open."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    System-level share target picker with an installed PWA as an option.
+  </figcaption>
+</figure>
+
+## See Web Share Target in action
+
+1. Using Chrome 76 or later, open the [Web Share Target demo][demo].
```

Comment:
I'm using v78, and the app is saying my browser doesn't support the Web Share API.

Also, should the app say "Web Share Target API" to avoid confusion?

Reply:
It's Android only right now. I've updated the text.